src/diffusers/pipelines/kandinsky2_2/pipeline_kandinsky2_2_inpainting.py

```diff
@@ -230,6 +230,8 @@ def prepare_mask_and_masked_image(image, mask, height, width):
         mask[mask >= 0.5] = 1
         mask = torch.from_numpy(mask)
 
+    mask = 1 - mask
```

Comment:
@yiyixuxu we need to do a breaking change here because Kandinsky can't accept a different mask format than SD. 

Let's make sure in the future that our APIs are always exactly aligned. We cannot have one pipeline having black pixels representing a mask and another having white pixels representing a mask.